systems/data-plan/base-rate/pkg/server/rate_test.go

```diff
@@ -0,0 +1,88 @@
+package server
```

Comment:
Good start. But these only test canonical/happy paths.  We can try to add sub tests for corner cases like trying to get a BaseRase that does not exist, et c...

Reply:
Addressed

Reply:
Yea, make sense. Addressed the feedback